### PR TITLE
Refactor rb side nav to use background images

### PR DESCRIPTION
**Pull request by gregbenner**
Refactor rb-side-nav to use background images instead of rb-badge

### CODE REVIEW

**Comment by gregbenner** [comment]
@ohskylab can you have a peak..?

File: test/unit/rb-side-nav/rb-side-nav-item.spec.js [hunk 1]
```diff
@@ -108,18 +108,6 @@ define([
             });
         });
 
-        describe('icon attribute', function () {
-
-            it('should use icon form icon attribute', function () {
-                compileTemplate('<rb-side-nav-item icon="blue-base-16-geo"></rb-side-nav-item>');
```
Comment (nwhite89) [inline test/unit/rb-side-nav/rb-side-nav-item.spec.js]: maybe instead of deleting this amend to ensure that it adds the correct class that it needs now?

Comment (gregbenner) [inline test/unit/rb-side-nav/rb-side-nav-item.spec.js]: Ah yeah can write a test, think @ohskylab removed this to pass the build step prior.

Comment (nwhite89) [inline test/unit/rb-side-nav/rb-side-nav-item.spec.js]: ok

Comment (ohskylab) [inline test/unit/rb-side-nav/rb-side-nav-item.spec.js]: Figured that test wasn't needed any more :)

Yes, should probably add one for new structure. @gregbenner, are you OK with adding?

**Comment by ohskylab** [comment]
LGTM. I think we need a matching PR in CM, otherwise this will break stuff when merged...

File: test/unit/rb-side-nav/rb-side-nav-item.spec.js (hunk 1 above)
Comment (gregbenner) [inline test/unit/rb-side-nav/rb-side-nav-item.spec.js]: Yep no problemo, adding now.

**Comment by gregbenner** [comment]
Pairs well with https://github.com/rockabox/rbx_campaign_management/pull/260

**Comment by ohskylab** [comment]
Historians: mentions of `rb-badge` should be `rb-icon`.

**Comment by gregbenner** [comment]
Ready to be accepted? :cactus:

**Comment by gregbenner** [comment]
#angular #reviewme

**Pull request merged**